artifactory/cli.go

```diff
@@ -1076,6 +1090,33 @@ func getReplicationDeleteFlags() []cli.Flag {
 	return getRepoDeleteFlags()
 }
 
+func getAccessTokenCreateFlags() []cli.Flag {
+	return append(getServerWithClientCertsFlags(), []cli.Flag{
+		cli.StringFlag{
+			Name: "groups",
+			Usage: "[Default: *] A list of comma-separated groups for the access token to be associated with. " +
+				"Specify * to indicate that this is a 'user-scoped token', i.e., the token provides the same access privileges that the current subject has, and is therefore evaluated dynamically. " +
+				"A non-admin user can only provide a scope that is a subset of the groups to which he belongs` `",
+		},
+		cli.BoolFlag{
+			Name:  "grant-admin",
+			Usage: "[Default: false] Set to true to provides admin privileges to the access token. This is only available for administrators.` `",
+		},
+		cli.StringFlag{
+			Name:  "expiry",
+			Usage: "[Default: " + strconv.Itoa(cliutils.TokenExpiry) + "] The time in seconds for which the token will be valid. To specify a token that never expires, set to zero. Non-admin can only set a value that is equal to or less than the default 3600.` `",
```

Comment:
Is this true?
Non-admin can only set a value that is equal to or less than the default 3600.
Can't a non-admin create an access token for week?